src/node.cc

```diff
@@ -2626,14 +2626,8 @@ void SetupProcessObject(Environment* env,
   READONLY_PROPERTY(process, "arch", OneByteString(env->isolate(), NODE_ARCH));
 
   // process.platform
-#ifdef _WIN32
-  // As determined by gyp, NODE_PLATFORM equals 'win' on windows. However
-  // for historic reasons process.platform should be 'win32'.
-  Local<String> platform = OneByteString(env->isolate(), "win32");
-#else
-  Local<String> platform = OneByteString(env->isolate(), NODE_PLATFORM);
-#endif
-  READONLY_PROPERTY(process, "platform", platform);
+  READONLY_PROPERTY(process, "platform",
+                                 OneByteString(env->isolate(), NODE_PLATFORM));
```

Comment:
@bnoordhuis what is the preferred way? Like this: https://github.com/iojs/io.js/blob/v1.x/src/node.cc#L2655 or like this https://github.com/iojs/io.js/blob/v1.x/src/node.cc#L2666 ?